lib/iiif/image_info_extractor.rb

```diff
@@ -2,16 +2,36 @@ module IIIF
   ##
   # Extracts a IIIF info.json-like hash for an image
   #
-  # @example 
+  # @example
   #   ImageInfoExtractor.extract(my_image) # => {:image => info}
   #
   class ImageInfoExtractor
+
+    autoload :Jp2Extractor, 'iiif/image_info_extractor/jp2_extractor'
+
+    CONTEXT_URI = "http://iiif.io/api/image/2/context.json"
+    COMPLIANCE_URI = "http://iiif.io/api/image/2/level2.json"
+    PROTOCOL_URI = "http://iiif.io/api/image"
+
     ##
     # @param [Image] image  an image to gather info for
     # @return [Hash] the image info
     def self.extract(image)
-      {}
+      info_seed = {
```

Comment:
This makes me wonder if we do want an ImageInfo object (to which we'd pass `image.path` and the extractor, I guess). Thoughts?